configuration/pih/liquibase/liquibase.xml

```diff
@@ -1118,5 +1118,45 @@
 
         </sql>
     </changeSet>
+
+
+    <changeSet id="20250325-void-duplicate-trauma-obs" author="ball">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                select count(*) from address_hierarchy_entry where parent_id is NULL and name like 'Haiti';
+            </sqlCheck>
+        </preConditions>
```

Comment:
wouldn't this run EVERY time for Haiti?  Should we put a count of the duplicate obs here in the precondition so it only runs when it needs to?